.gitignore

```diff
@@ -1,2 +1,4 @@
+
+**/.DS_Store
```

Comment:
Typically you would put `*.DS_Store` in your global gitignore so that it's ignored in all of your git repos automatically